frame/App/Body/Main/Layout/components/ListItem.js

```diff
@@ -62,6 +65,8 @@ const ListItem = ({
           />
         </button>
       )}
+      {priority ? <Icon icon="flag-variant" /> : null}
+
```

Comment:
You can also do this which makes the code easier to read. it's called short circuiting
```suggestion
      {priority && <Icon icon="flag-variant" />}
```